src/client/sandbox/native-methods.js

```diff
@@ -59,6 +59,18 @@ class NativeMethods {
 
         const documentCookieDescriptor = win.Object.getOwnPropertyDescriptor(win[this.documentCookiePropOwnerName].prototype, 'cookie');
 
+        if (documentCookieDescriptor.get.toString().indexOf('native code') === -1) {
```

Comment:
Do we have util method for check that function is native?

Reply:
No, and it unnecessary because it temporary solution